python/lsst/pipe/tasks/diff_matched_tract_catalog.py

```diff
@@ -107,18 +108,23 @@ class DiffMatchedTractCatalogConnections(
         dimensions=("tract", "skymap"),
     )
     cat_matched = cT.Output(
-        doc="Catalog with reference and target columns for matched sources only",
+        doc="Catalog with reference and target columns for joined sources",
```

Comment:
Why the switch from "matched" to "joined" terminology?

Reply:
After adding the option to output unmatched rows, the table no longer necessarily contains only matched objects.